src/sftp.c

```diff
@@ -357,24 +350,6 @@ sftp_packet_read(LIBSSH2_SFTP *sftp)
             sftp->partial_received = 0; /* how much of the packet already
                                            received */
             sftp->partial_packet = packet;
-
-          window_adjust:
```

Comment:
Ditto my previous comment about window adjustment in _libssh2_channel_read.
@bagder This was originally introduced in your commit 7194a9bd7ba454b00b00f67ee7c0597a637b3c36